Userland/Utilities/rev.cpp

```diff
@@ -25,29 +27,43 @@ int main(int argc, char** argv)
     args_parser.parse(argc, argv);
 
     Vector<RefPtr<Core::File>> files;
-    if (paths.is_empty()) {
-        files.append(Core::File::standard_input());
```

Comment:
Your commit message says
>With this commit, if no files are supplied to rev, it will by default read user input from stdin.

But that was already accomplished by the code you are removing here, no?

Reply:
Or oh, I see. The old code doesn't work correctly.
Well, we should make it work correctly instead of making separate logic for stdin vs any other file. :)

Reply:
Yeah, it seems that appending `Core::File::standard_input()` causes it to terminate immediately if you don't supply any files. It also causes it to *sometimes* work when piping in input (when I originally tested it I got lucky it seems), however further testing it, it seems that more often then not it won't work and will terminate instantly.